### ISSUE TITLE
App crashes when searching on `crossw`

### ISSUE THREAD

**Issue by joeb37** [CS599-MEDLINE/pubminer-ui-investigation#57]
1) Search for `crossw`
2) The following gets logged to the console:
```
calling esearch for crossw...
processing esearch result for (((randomized controlled trial[pt]) OR (controlled clinical trial[pt]) OR (randomized[tiab] OR randomised[tiab]) OR (placebo[tiab]) OR (drug therapy[sh]) OR (randomly[tiab]) OR (trial[tiab]) OR (groups[tiab])) NOT (animals[mh] NOT humans[mh])) AND (crossw)
calling elink...
processing elink result
error performing search TypeError: Cannot read property 'linksets' of undefined
Unhandled rejection TypeError: Cannot read property 'uids' of undefined
    at pmSvc.fetchSummary.then.summaryResults (/Users/Joe/Developer/cscie599/pmc-oa-test/search.js:81:72)
    at tryCatcher (/Users/Joe/Developer/cscie599/pmc-oa-test/node_modules/bluebird/js/release/util.js:16:23)
    at Promise._settlePromiseFromHandler (/Users/Joe/Developer/cscie599/pmc-oa-test/node_modules/bluebird/js/release/promise.js:512:31)
    at Promise._settlePromise (/Users/Joe/Developer/cscie599/pmc-oa-test/node_modules/bluebird/js/release/promise.js:569:18)
    at Promise._settlePromise0 (/Users/Joe/Developer/cscie599/pmc-oa-test/node_modules/bluebird/js/release/promise.js:614:10)
    at Promise._settlePromises (/Users/Joe/Developer/cscie599/pmc-oa-test/node_modules/bluebird/js/release/promise.js:693:18)
    at Async._drainQueue (/Users/Joe/Developer/cscie599/pmc-oa-test/node_modules/bluebird/js/release/async.js:133:16)
    at Async._drainQueues (/Users/Joe/Developer/cscie599/pmc-oa-test/node_modules/bluebird/js/release/async.js:143:10)
    at Immediate.Async.drainQueues [as _onImmediate] (/Users/Joe/Developer/cscie599/pmc-oa-test/node_modules/bluebird/js/release/async.js:17:14)
    at runCallback (timers.js:757:18)
    at tryOnImmediate (timers.js:718:5)
    at processImmediate [as _immediateCallback] (timers.js:698:5)
    at process.topLevelDomainCallback (domain.js:101:23)
```

**Comment by freshottr** [comment]
It looks like we aren't handling empty search results properly. I'll have a look.

**Comment by freshottr** [comment]
A search for misspelled `guillane barre` reproduces this issue. 
```
calling esearch for (guillane barre) AND (((randomized controlled trial[pt]) OR (controlled clinical trial[pt]) OR (randomized[tiab] OR randomised[tiab]) OR (placebo[tiab]) OR (drug therapy[sh]) OR (randomly[tiab]) OR (trial[tiab]) OR (groups[tiab])) NOT (animals[mh] NOT humans[mh]))...
processing esearch result for (guillane barre) AND (((randomized controlled trial[pt]) OR (controlled clinical trial[pt]) OR (randomized[tiab] OR randomised[tiab]) OR (placebo[tiab]) OR (drug therapy[sh]) OR (randomly[tiab]) OR (trial[tiab]) OR (groups[tiab])) NOT (animals[mh] NOT humans[mh]))
calling elink...
processing elink result
error performing search TypeError: Cannot read property 'linksets' of undefined
results.js calling getSummaries {"error":"unexpected error performing search"}
unexpected error processing getSummaries TypeError: Cannot read property 'uids' of undefined TypeError: Cannot read property 'uids' of undefined
    at pmSvc.fetchSummary.then.summaryResults (/Users/ismith/school/cs599/pubminer-ui-investigation/search.js:90:72)
    at tryCatcher (/Users/ismith/school/cs599/pubminer-ui-investigation/node_modules/bluebird/js/release/util.js:16:23)
    at Promise._settlePromiseFromHandler (/Users/ismith/school/cs599/pubminer-ui-investigation/node_modules/bluebird/js/release/promise.js:512:31)
    at Promise._settlePromise (/Users/ismith/school/cs599/pubminer-ui-investigation/node_modules/bluebird/js/release/promise.js:569:18)
    at Promise._settlePromise0 (/Users/ismith/school/cs599/pubminer-ui-investigation/node_modules/bluebird/js/release/promise.js:614:10)
    at Promise._settlePromises (/Users/ismith/school/cs599/pubminer-ui-investigation/node_modules/bluebird/js/release/promise.js:693:18)
    at Async._drainQueue (/Users/ismith/school/cs599/pubminer-ui-investigation/node_modules/bluebird/js/release/async.js:133:16)
    at Async._drainQueues (/Users/ismith/school/cs599/pubminer-ui-investigation/node_modules/bluebird/js/release/async.js:143:10)
    at Immediate.Async.drainQueues [as _onImmediate] (/Users/ismith/school/cs599/pubminer-ui-investigation/node_modules/bluebird/js/release/async.js:17:14)
    at runCallback (timers.js:763:18)
    at tryOnImmediate (timers.js:734:5)
    at processImmediate (timers.js:716:5)
    at process.topLevelDomainCallback (domain.js:102:23)
TypeError: /Users/ismith/school/cs599/pubminer-ui-investigation/views/results.pug:10
    8|     script(src="scripts/pubminer.js")
    9| block content
  > 10|     if results.items.length == 0
    11|         p(class='results-summary') No results found for "#{qryResult.searchTerm}"
    12|     else
    13|         input(type="hidden" id="webenv" value=qryResult.webenv)

Cannot read property 'length' of undefined
    at eval (eval at wrap (/Users/ismith/school/cs599/pubminer-ui-investigation/node_modules/pug-runtime/wrap.js:6:10), <anonymous>:164:19)
    at template (eval at wrap (/Users/ismith/school/cs599/pubminer-ui-investigation/node_modules/pug-runtime/wrap.js:6:10), <anonymous>:650:72)
    at Object.exports.renderFile (/Users/ismith/school/cs599/pubminer-ui-investigation/node_modules/pug/lib/index.js:427:38)
    at Object.exports.renderFile (/Users/ismith/school/cs599/pubminer-ui-investigation/node_modules/pug/lib/index.js:417:21)
    at View.exports.__express [as engine] (/Users/ismith/school/cs599/pubminer-ui-investigation/node_modules/pug/lib/index.js:464:11)
    at View.render (/Users/ismith/school/cs599/pubminer-ui-investigation/node_modules/express/lib/view.js:135:8)
    at tryRender (/Users/ismith/school/cs599/pubminer-ui-investigation/node_modules/express/lib/application.js:640:10)
    at Function.render (/Users/ismith/school/cs599/pubminer-ui-investigation/node_modules/express/lib/application.js:592:3)
    at ServerResponse.render (/Users/ismith/school/cs599/pubminer-ui-investigation/node_modules/express/lib/response.js:1008:7)
    at pubMedQuery.getSummaries (/Users/ismith/school/cs599/pubminer-ui-investigation/routes/results.js:22:26)
    at pmSvc.fetchSummary.then.catch.err (/Users/ismith/school/cs599/pubminer-ui-investigation/search.js:117:17)
    at tryCatcher (/Users/ismith/school/cs599/pubminer-ui-investigation/node_modules/bluebird/js/release/util.js:16:23)
    at Promise._settlePromiseFromHandler (/Users/ismith/school/cs599/pubminer-ui-investigation/node_modules/bluebird/js/release/promise.js:512:31)
    at Promise._settlePromise (/Users/ismith/school/cs599/pubminer-ui-investigation/node_modules/bluebird/js/release/promise.js:569:18)
    at Promise._settlePromise0 (/Users/ismith/school/cs599/pubminer-ui-investigation/node_modules/bluebird/js/release/promise.js:614:10)
    at Promise._settlePromises (/Users/ismith/school/cs599/pubminer-ui-investigation/node_modules/bluebird/js/release/promise.js:689:18)
    at Async._drainQueue (/Users/ismith/school/cs599/pubminer-ui-investigation/node_modules/bluebird/js/release/async.js:133:16)
    at Async._drainQueues (/Users/ismith/school/cs599/pubminer-ui-investigation/node_modules/bluebird/js/release/async.js:143:10)
    at Immediate.Async.drainQueues (/Users/ismith/school/cs599/pubminer-ui-investigation/node_modules/bluebird/js/release/async.js:17:14)
    at runCallback (timers.js:763:18)
    at tryOnImmediate (timers.js:734:5)
    at processImmediate (timers.js:716:5)
```

**Comment by freshottr** [comment]
should be addressed by #81
